opentech/apply/users/migrations/0014_add_sentinel_user.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.0.13 on 2019-09-03 15:15
+
+from django.db import migrations
+
+
+SENTINEL_EMAIL = 'sentinel'
+
+
+def add_sentinel_user(apps, schema_editor):
+    User = apps.get_model('users', 'User')
+    User.objects.create(
+        email=SENTINEL_EMAIL,
+        full_name='sentinel',
```

Comment:
I was expecting the sentinal to be specific to the use case. e.g. "Compliance", or similar so that when we log the activity we have a sensible public facing name for the activity log.

Reply:
Fixed and rebased.